src/Razor/src/Microsoft.AspNetCore.Razor.LanguageServer.Common/LanguageServerConstants.cs

```diff
@@ -51,6 +53,15 @@ public static class CodeActions
             public const string CreateComponentFromTag = "CreateComponentFromTag";
 
             public const string AddUsing = "AddUsing";
+
+            public const string Default = "Default";
```

Comment:
What's a default code action?

Reply:
The Default (CSharp) code action essentially just acts as a proxy (resolving the code action through LSP/Roslyn) and remapping/formatting, without any other razor specific post-processing. An example of razor specific post-processing is adding a `@` to `using System.IO;` provided by Roslyn to make `@using System.IO`.

Reply:
Got it. How will this change when, for example, the HTML language server started supporting some code actions that we also want to support?

Reply:
We essentially have a "double key" system. The code action resolver is identified by `Languages.CodeActions` (ex. `CSharp.Default`). In HTML's case, should we need something similar, we'd probably be implementing a `HTML.Default` resolver which handles the default (HTML specific case). Ex. things like formatting and such would change with HTML.